src/activity_handlers.cpp

```diff
@@ -654,7 +654,7 @@ int butcher_time_to_cut( const player &u, const item &corpse_item, const butcher
     const mtype &corpse = *corpse_item.get_mtype();
     const int factor = u.max_quality( action == butcher_type::DISSECT ? qual_CUT_FINE : qual_BUTCHER );
 
-    int time_to_cut = 0;
+    int time_to_cut;
```

Comment:
we could also just initialize it to 450 here, but i think the intent is clearer the way i have it now